source/sdk/math.ooc

```diff
@@ -59,6 +59,10 @@ floor: extern(floorl) func ~Long (LDouble) -> LDouble
 	 maximumValue ::= static SHRT_MAX
 }
 
+extend Int64 {
+	modulo: func(divisor: Int) -> This { this - divisor * (this / divisor) }
+}
```

Comment:
1. It differs in how it handles negative numbers, if that's what you mean. Yes, we need to decide that. Guess that discussion belongs to #508 
2. Yep, you're right, will fix.